src/main/java/org/example/models/SingleJobRoleResponse.java

```diff
@@ -0,0 +1,20 @@
+package org.example.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.sql.Date;
+
+@AllArgsConstructor @Getter @Setter
+public class SingleJobRoleResponse {
```

Comment:
what's a single job role response? How is it different to a jobroleresponse? 

Adding single to a class name seems like an antipattern. We have lists / collections to make things multiple. I would double check this, maybe it's just a naming thing? Maybe JobRoleDetailsResponse?

Reply:
a SingleJobRoleResponse has the same fields as the JobRoleResponse except it also has the description, responsibilities, and job link and it doesn't have the role_id. Would it be okay to just rename it to JobRoleDetailsResponse?

Reply:
JobRoleDetailsResponse makes sense to me :)

Reply:
fixed!